tracer/src/Datadog.AutoInstrumentation.NativeLoader/instrumented_assembly_generator/member_signature.cpp

```diff
@@ -0,0 +1,235 @@
+#include "member_signature.h"
+#include "instrumented_assembly_generator_helper.h"
+
+namespace instrumented_assembly_generator
+{
+namespace
+{
```

Comment:
why putting those function in an unnamed namespace?

Reply:
Fixed in ebde2f57c8ef03fd167862f981abd5c4958c222a